test/tests/functional/pbs_pbsnodes.py

```diff
@@ -55,10 +56,17 @@ def setUp(self):
         self.hostA = self.moms.values()[0].shortname
 
     def common_setUp(self):
+        """
+        Common setUp for tests test_pbsnodes_as_user and test_pbsnodes_as_root
+        """
         TestFunctional.setUp(self)
         platform = self.du.get_platform()
         if platform == 'cray' or platform == 'craysim':
             self.skipTest("cannot run on cray or cray_sim system")
```

Comment:
I have same question, why not using skiponcray decorator for this?

Reply:
I changed the code to use decorator skipOnCray.